.github/workflows/ci.yml

```diff
@@ -47,52 +47,14 @@ jobs:
     name: Push charts to the web
     runs-on: ubuntu-latest
     steps:
-      - name: stunner checkout
-        uses: actions/checkout@v3
-        with:
-          path: stunner-gateway-operator
-          ref: main
-          repository: l7mp/stunner-gateway-operator
       - name: Get the version
         id: vars
         run: echo ::set-output name=tag::$(echo ${GITHUB_REF:11})
-      - name: l7mp.io checkout
-        uses: actions/checkout@v3
+      - name: Trigger repository dispatch event with passing the tagged version
+        uses: peter-evans/repository-dispatch@v2.1.1
```

Comment:
I recommend to fix the major version only to enable potential fixes in the later minor/patch releases.
```diff
+        uses: peter-evans/repository-dispatch@v2
-        uses: peter-evans/repository-dispatch@v2.1.1
```